hip-clang.docker

```diff
@@ -31,6 +31,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-
     libnuma-dev \
     miopen-hip \
     rocblas \
+    hipblaslt \
```

Comment:
This should be installed in the install_preqs.sh file, not in the dockerfile.

Reply:
Moved to install_prereqs file.